jobserver/api/jobs.py

```diff
@@ -118,6 +118,8 @@ def post(self, request, *args, **kwargs):
                     )
 
                     span.set_attribute("job_id", str(job.id))
```

Comment:
If we're going to keep it, I think we should call this `django_id` or `db_id`. It's internal to job-server db and should never be used externallly.